### PR TITLE
Grimbly Station mass cherry-pick

### PR DESCRIPTION
**Pull request by Tmanzxd**
# Description

Adds all the following PRs from Grimbly Station:

Allow Bladed Melee Weapons to Cut Food: https://github.com/Grimbly-Station/Grimbly-Station/pull/132

Insuzine no longer takes benzene: https://github.com/Grimbly-Station/Grimbly-Station/pull/127

Lowers antag amount for Pop Balance: https://github.com/Grimbly-Station/Grimbly-Station/pull/124

HoP Access Rework: https://github.com/Grimbly-Station/Grimbly-Station/pull/99

Make Rat King more Neutral: https://github.com/Grimbly-Station/Grimbly-Station/pull/96

consumption is important: https://github.com/Grimbly-Station/Grimbly-Station/pull/129

Emergency Toolbox Rework: https://github.com/Grimbly-Station/Grimbly-Station/pull/83

Quickswap enable by default: https://github.com/Grimbly-Station/Grimbly-Station/pull/77

Make CE Hardsuit Fireproof: https://github.com/Grimbly-Station/Grimbly-Station/pull/76

Re-adds Fireaxe Prying: https://github.com/Grimbly-Station/Grimbly-Station/pull/73

# Changelog

:cl:
- add: Added HoP Accesses like from Goobstation
- add: Bladed weapons cutting food.
- remove: Benzene from Insuzine chem recipe.
- tweak: Lowered Antag amount for lowpop balance purposes.
- tweak: Encourage more neutral gameplay for Rat Kings. They can wear headsets now.
- add: Bring back autoeat and autodrink doAfters.
- add: /tg/station Emergency Toolbox Rework.
- add: Make CE Hardsuit Fireproof.
- add: re-add Fire Axe prying.

## Summary by CodeRabbit

- **New Features**
	- Automatic eating and drinking actions are now enabled.
	- Quick swapping of item slots is now possible.
	- New items added to toolboxes, including a pocket fire extinguisher and various lighting items.
	- New inventory template for the "Rat King" role introduced.
	- New component added to melee weapons to allow dual functionality as utensils.
	- Enhanced protective capabilities for the Chief Engineer's Hardsuit with temperature protection.
	- Added stamina damage resistance to the Security Hardsuit.

- **Bug Fixes**
	- Adjusted maximum player limit for the Traitor role to enhance game balance.

- **Documentation**
	- Updated role descriptions and comments for various entities to reflect changes.

- **Style**
	- Renamed "Brigmedic Hardsuit" to "Corpsman Hardsuit" for clarity.

- **Chores**
	- Introduced metadata for the new pocket fire extinguisher resource.

### CODE REVIEW

**Comment by Tmanzxd** [comment]
MobRatKing ID duplicate.

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 1]
```diff
@@ -7,24 +7,29 @@
...
-      - id: HarmonicaInstrument
+      # Low-chance items
+      - id: FoodSnackChocolate
         prob: 0.15
-
+      - id: HarmonicaInstrument
+        prob: 0.10
+      - id: FoodTinMRE
+        prob: 0.20
+
```
[the remark this answers is not in the document]
Comment (Tmanzxd) [reply Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:32]: The space is for the next entity in the YML list.

File: Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml [hunk 2]
```diff
@@ -84,3 +84,36 @@
...
+  name: pocket fire extinguisher
+  parent: FireExtinguisher
+  id: FireExtinguisherMini
+  description: A light and compact fibreglass-framed model fire extinguisher. It holds less water then its bigger brother.
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/fire_extinguisher_mini.rsi
+    layers:
+        - state: fire_extinguisher_closed
+          map: [ "enabled" ]
```
[the remark this answers is not in the document]
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml:98]: Indentation is correct.

**Comment by Tmanzxd** [comment]
MobRatKing ID duplicate. Will look into it later.

File: Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl [hunk 3]
```diff
@@ -52,7 +52,7 @@ ghost-role-information-kobold-description = Be the little gremlin you are, yell
 
 ghost-role-information-rat-king-name = Rat King
 ghost-role-information-rat-king-description = You are the Rat King, scavenge food in order to produce rat minions to do your bidding.
-ghost-role-information-rat-king-rules = You are an antagonist, scavenge, attack, and grow your hoard!
+ghost-role-information-rat-king-rules =  You have no allegiance, so scavenge, attack, and grow your hoard or learn to live in peace with the crew.
```
Comment (angelofallars) [inline Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl:55]: add a comment above this line with `# Grimbly - modify rat king rules to be more neutral`

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 4]
```diff
@@ -7,24 +7,29 @@
   - type: StorageFill
     contents:
       - id: CrowbarRed
+      - id: RadioHandheld
```
Comment (angelofallars) [inline Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:10]: Mark all new items with something like `# Grimbly items start` `# Grimbly items end` I think

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 5]
```diff
@@ -375,6 +375,8 @@
     lowPressureMultiplier: 1000
   - type: ExplosionResistance
     damageCoefficient: 0.2
+  - type: TemperatureProtection
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:378]: Add a `# Grimbly` comment to this

File: Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
 - type: entity
   name: Rat King
   id: MobRatKing
-  parent: [ SimpleMobBase, MobCombat ]
-  description: He's da rat. He make da roolz.
+  parent: [ SimpleMobBase, MobCombat ] 
+  description: He's da rat king. He make da roolz.
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml:5]: Add a `# Grimbly` comment here to indicate that the description changed

File: Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml [hunk 7]
```diff
@@ -42,6 +42,19 @@
         - SmallMobMask
         layer:
         - SmallMobLayer
+  - type: Hands #Grimbly - ratking ID and headset
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml:45]: A `# Grimbly start` `# Grimbly end` would go hard here I think

File: Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml [hunk 8]
```diff
@@ -84,3 +84,36 @@
   - type: VaporVisuals
     animationTime: 0.8
     animationState: extinguish
+
+- type: entity
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml:88]: ~~Add a `# Grimbly` comment here~~

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 9]
```diff
@@ -557,19 +557,6 @@
   products:
     Insuzine: 3
 
-- type: reaction
```
Comment (angelofallars) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:560]: Why was Opporozidone deleted?

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 10]
```diff
@@ -74,7 +75,9 @@
     back: ClothingBackpackHOPFilled
     shoes: ClothingShoesLeather # DeltaV - HoP needs something better than plebe shoes.
     id: HoPPDA
-    ears: ClothingHeadsetHoP # DeltaV - HoP is now a service role, replaces their all channels headset.
+    gloves: ClothingHandsGlovesHop
```
Comment (angelofallars) [inline Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:78]: Delete the gloves and belt here, they were removed in EE for the Great Loadout Reorganizing merged November 18, 2024. [6c213256d6dcb35954593af0e950db3a2cb07910](https://github.com/SpaciousStation14/SpaciousStation14/commit/6c213256d6dcb35954593af0e950db3a2cb07910#diff-961c2f17284205fa71577c5f4f4325d31de65ae28445259e7608e31ee4a9ac06)

File: Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml (hunk 8 above)
Comment (angelofallars) [reply Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml:88]: Don't think a # Grimbly comment is necessary here, we can just upstream the mini fire extinguisher to EE soon^tm

**Review by angelofallars — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Reactions/medicine.yml (hunk 9 above)
Comment (Tmanzxd) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml:560]: Unsure. Reverted it.

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 11]
```diff
@@ -70,11 +71,13 @@
 - type: startingGear
   id: HoPGear
   equipment:
-    jumpsuit: ClothingUniformJumpsuitHoP
+    jumpsuit: ClothingUniformJumpsuitHop
```
Comment (angelofallars) [inline Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:74]: accidentally turned `ClothingUniformJumpsuitHoP` into `ClothingUniformJumpsuitHop` here, last letter should be uppercase P

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml (hunk 10 above)
Comment (angelofallars) [reply Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:78]: gloves and belt are still here

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 12]
```diff
@@ -584,6 +571,19 @@
   products:
     Necrosol: 2
 
+- type: reaction
```
Comment (angelofallars) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:574]: put Opporozidone above Necrosol which is where it was placed before, to revert the change entirely, it's still a thing that will cause merge conflicts

File: Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml [hunk 13]
```diff
@@ -42,11 +42,24 @@
         - SmallMobMask
         layer:
         - SmallMobLayer
+  - type: Hands #Grimbly - ratking ID, headset, hands - Start
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml:45]: Nitpick: for block-based indicator comments with start and end, put the comment on a separate line. here the start comment should be above the Hands definition, and the end should be below the MobThresholds definition

File: Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl (hunk 3 above)
Comment (angelofallars) [reply Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl:56]: ^ comment should be right above `ghost-role-information-rat-king-rules`

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 14]
```diff
@@ -7,24 +7,31 @@
...
+      - id: WelderMini
+      - id: FireExtinguisherMini
+      # Random lighting item orGroup
+      - id: FlashlightLantern
+        orGroup: LightingItem
+      - id: Flare
+        orGroup: LightingItem
+      - id: GlowstickBase
+        orGroup: LightingItem
+      # Grimbly - Oxygen
```
Comment (angelofallars) [inline Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:21]: replace `# Grimbly - Oxygen` with `# Grimbly items end`

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 15]
```diff
@@ -7,24 +7,31 @@
...
-      - id: FoodTinMRE
-        prob: 0.5
-      - id: DrinkWaterBottleFull
-      - id: DrinkWaterBottleFull
-        prob: 0.5
-      - id: FlashlightLantern
-      - id: FlashlightLantern
-        prob: 0.5
-      - id: HarmonicaInstrument
+      # Grimbly items end
```
Comment (angelofallars) [inline Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:26]: Replace `# Grimbly items end` with `# Grimbly low-chance items start`

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 16]
```diff
@@ -7,24 +7,31 @@
...
+      # Low-chance items
+      - id: FoodSnackChocolate
         prob: 0.15
-
+      - id: HarmonicaInstrument
+        prob: 0.10
+      - id: FoodTinMRE
+        prob: 0.20
```
Comment (angelofallars) [inline Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:33]: add comment in the line below `# Grimbly low-chance items end`

File: Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml [hunk 17]
```diff
@@ -1,8 +1,8 @@
 - type: entity
   name: Rat King
   id: MobRatKing
-  parent: [ SimpleMobBase, MobCombat ]
-  description: He's da rat. He make da roolz.
+  parent: [ SimpleMobBase, MobCombat ]
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Mobs/NPCs/regalrat.yml:4]: remove the leading space here after the ]

**Review by angelofallars — CHANGES_REQUESTED** [review]

**Review by angelofallars — APPROVED** [review]

**Comment by angelofallars** [comment]
https://github.com/SpaciousStation14/SpaciousStation14/pull/6/commits/6feacdac5b97a667a73b0f3ae071f99781466c5a

@Tmanzxd  Rat King bug was because `type: UserInterface` was declared twice, first in Grimbly and the second time was when Shitmed was added to add the surgery UI to the rat king.

**Pull request merged**